packages/ckeditor5-code-block/src/codeblockui.ts

```diff
@@ -27,21 +28,21 @@ export default class CodeBlockUI extends Plugin {
 	/**
 	 * @inheritDoc
 	 */
-	static get pluginName() {
+	public static get pluginName(): string {
```

Comment:
```suggestion
	public static get pluginName(): 'CodeBlockUI' {
```

Reply:
And add this to `PluginsMap` (all plugins should be registered in `PluginsMap` type).